### PR TITLE
fix(macOS) TextBox Vertical Padding

### LINKED ISSUE #4740: Alignment issues & height issues for `TextBox` on macOS
## Current behavior

The alignment and sizing of `TextBox`-based controls on macOS differs from other targets.

## Expected behavior

Sizing should match other platforms.

## How to reproduce it (as minimally and precisely as possible)

Clone https://github.com/MartinZikmund/Uno.ConwaysGameOfLife and run on macOS. Notice the alignment of controls below buttons does not match other platforms.

## Environment

Nuget Package:

- [x] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [x] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #4740

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?
Vertical Padding on macOS TextBox is incorrect

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml [hunk 1]
```diff
@@ -1493,7 +1493,7 @@
       <x:Boolean x:Key="UseSystemFocusVisuals">True</x:Boolean>
       <Thickness x:Key="TextControlBorderThemeThickness">1</Thickness>
       <Thickness x:Key="TextControlBorderThemeThicknessFocused">2</Thickness>
-      <Thickness x:Key="TextControlThemePadding">10,6,6,5</Thickness>
+      <macos:Thickness x:Key="TextControlThemePadding">10,8,6,5</macos:Thickness>
```
Comment (jeromelaban) [inline src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml:1496]: Should you be adding a `not_macos` as well here ?

File: src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml [hunk 2]
```diff
@@ -1726,7 +1726,7 @@
       <x:Boolean x:Key="UseSystemFocusVisuals">True</x:Boolean>
       <Thickness x:Key="TextControlBorderThemeThickness">1</Thickness>
       <Thickness x:Key="TextControlBorderThemeThicknessFocused">1,1,1,2</Thickness>
-      <Thickness x:Key="TextControlThemePadding">10,5,6,6</Thickness>
+      <macos:Thickness x:Key="TextControlThemePadding">10,8,6,5</macos:Thickness>
```
Comment (jeromelaban) [inline src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml:1729]: Same `not_macos` for this file

File: src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml (hunk 1 above)
Comment (ajpinedam) [reply src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml:1496]: The tricky part is that I did not modify this file (nor the other one). I updated these two:

```
src/Uno.UI.FluentTheme.v1/Resources/Version1/Priority01/Common_themeresources_v1.xaml
src/Uno.UI.FluentTheme.v2/Resources/Version2/Priority01/Common_themeresources.xaml
```

and when I compiled these two were updated

````
src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml
src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml
````

Should I manually add the values to src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml? I will try and recompile to see how it goes

Comment (jeromelaban) [reply src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml:1496]: Let's try to make another PR that changes this block: https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v2/FluentMerge.targets#L10-L39

to use https://github.com/unoplatform/uno.xamlmerge.task instead.

**Comment by jeromelaban** [comment]
Let's skip AppKit updates for now.

**Pull request closed without merging**